frontend/src/routes/manage/Realm/AddChild.tsx

```diff
@@ -44,7 +44,12 @@ export const AddChildRoute = makeRoute(url => {
     return {
         render: () => <RootLoader
             {...{ query, queryRef }}
-            nav={data => data.parent ? <Nav fragRef={data.parent} /> : []}
+            nav={data => data.parent
+                ? [
+                    <Nav key="main-nav" fragRef={data.parent} />,
+                    <RealmEditLinks key="edit-buttons" path={parent} />,
+                ]
```

Comment:
Btw can we replace this with fragments somehow? :thinking:

Reply:
We could, but right now we specifically model it as `[] | JSX.Element | [JSX.Element, JSX.Element]` because we only support those cases. Three items would break stuff. So I would like to keep it as is.

Reply:
Fair.